x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/expandable_event.tsx

```diff
@@ -123,24 +123,32 @@ export const ExpandableEventTitle = React.memo<ExpandableEventTitleProps>(
             </>
           )}
         </EuiFlexItem>
-        {handleOnEventClosed && (
-          <EuiFlexItem grow={false}>
-            <EuiButtonIcon iconType="cross" aria-label={i18n.CLOSE} onClick={handleOnEventClosed} />
-          </EuiFlexItem>
-        )}
-        {isAlert && (
-          <EuiCopy textToCopy={alertDetailsLink}>
-            {(copy) => (
-              <EuiButtonEmpty
-                onClick={copy}
-                iconType="share"
-                data-test-subj="copy-alert-flyout-link"
-              >
-                {i18n.SHARE_ALERT}
-              </EuiButtonEmpty>
+        <EuiFlexItem grow={false}>
```

Comment:
This was a visual bug fix for the position of the `Share Alert Details` button here: https://github.com/elastic/kibana/issues/155030

Reply:
Not for this PR, but while desk testing, we noted that:

- URLs generated by the `kibana.alert.url` feature in this PR do NOT encode the `:` characters in the `&timestamp=2023-04-24T18:10:36.225Z` URL parameter, per the example below:

```
http://localhost:5601/s/space-madness/app/security/alerts/redirect/aa7c1d26c66a7a549233c000bc8c240979a273abad6b50ada5b19f7dec86553a?index=.alerts-security.alerts-space-madness&timestamp=2023-04-24T18:10:36.225Z
```

_Above: A URL generated with the `kibana.alert.url` feature in this PR_

However, URLs generated by the `Share Alert Deatils` button encode the `:` characters in the `&timestamp=2023-04-24T18%3A10%3A36.225Z` URL parameter, per the example below:

```
http://localhost:5601/s/space-madness/app/security/alerts/redirect/aa7c1d26c66a7a549233c000bc8c240979a273abad6b50ada5b19f7dec86553a?index=.alerts-security.alerts-space-madness&timestamp=2023-04-24T18%3A10%3A36.225Z
```

_Above: A URL for the same alert, generated with the `Share Alert Details` button_

Since it doesn't feel necessary to encode the `:` characters, the differences in behavior may be worth investigating / normalizing.